apps/eth_blockchain/contracts/README.md

```diff
@@ -0,0 +1,22 @@
+## Contract sources
```

Comment:
Looks like there isn't a way to track the OpenZepplin version we're using. It'll be hard to track as time passes. Should we have a `OPENZEPPLIN_VERSION` file in this folder that has something like this?

```
v2.3.0
```